### ISSUE TITLE
Browsing a gophernicus served directory with no gophermap file

### ISSUE THREAD

**Issue by DavidGriffith** [jgoerzen/gopher#2]
I originally raised this issue at https://github.com/gophernicus/gophernicus/issues/48 whereupon the finger was pointed at UM Gopher.  After some investigations, I'm even less clear on where the problem actually exists.

Now the bug, looking at gopher://gopher.661.org/, when you try to view or download any file, the filename you'll get will look something like this:
````
dfrotz.zip----------------------------2015-Oct-27-
```` 

If there is a `gophermap` file in the directory, then this doesn't happen.

**Comment by DavidGriffith** [comment]
When a directory containing a `gophermap` file is requested, this is what is returned by Gophernicus:

````
telnet gopher.661.org 70
Trying 192.81.131.210...
Connected to hermes.661.org.
Escape character is '^]'.
/vx7/
iConfig files for the Yaesu VX-7R HT ham radio		null.host	1
i		null.host	1
iA blank config file.		null.host	1
9blank-vx7.img	/vx7/blank-vx7.img	gopher.661.org	70
i		null.host	1
i2m/1.25m/70cm repeaters in Bakersfield, CA		null.host	1
9bakersfield.img	/vx7/bakersfield.img	gopher.661.org	70
i		null.host	1
i2m/1.25m/70cm repeaters in Bellingham, WA		null.host	1
9bellingham.img	/vx7/bellingham.img	gopher.661.org	70
i		null.host	1
i2m/1.25m/70cm repeaters in Seattle, WA		null.host	1
9seattle.img	/vx7/seattle.img	gopher.661.org	70
i		null.host	1
iFRS and GMRS frequencies		null.host	1
9frs-gmrs.img	/vx7/frs-gmrs.img	gopher.661.org	70
i___________________________________________________________________		null.host	1
i                 Gophered by Gophernicus/3.0.1 on Debian/10 x86_64		null.host	1
.
````

This is what is returned without a `gophermap` file:
````
telnet gopher.661.org 70
/text/
Trying 192.81.131.210...
Connected to hermes.661.org.
Escape character is '^]'.
i[/text/]	TITLE	null.host	1
i		null.host	1
1..                                                                	/	gopher.661.org	70
0lily-the-pink.txt                     2013-Feb-25 07:32     2.3 KB	/text/lily-the-pink.txt	gopher.661.org	70
0pants.txt                             2002-Dec-18 15:29     2.1 KB	/text/pants.txt	gopher.661.org	70
0printcrime.txt                        2013-Feb-19 13:28     4.3 KB	/text/printcrime.txt	gopher.661.org	70
0right-to-read.txt                     2003-Jan-07 23:41     9.8 KB	/text/right-to-read.txt	gopher.661.org	70
0rpg-rules.txt                         2008-Mar-19 10:43    17.5 KB	/text/rpg-rules.txt	gopher.661.org	70
i___________________________________________________________________		null.host	1
i                 Gophered by Gophernicus/3.0.1 on Debian/10 x86_64		null.host	1
.
````

The most noticeable difference between the two is that for the `gophermap`-controlled directory, the selector string is separated from the rest of the line with tab characters.  For the directory without a `gophermap` file, the separators are spaces.  According to how I'm interpreting https://www.w3.org/Addressing/URL/4_1_Gopher+.html, spaces may not be used like this.

Opinions?

**Comment by DavidGriffith** [comment]
Closing because the problem is fixed by #3.
